classes/migration/upgrade/PKPv3_4_0UpgradeMigrationTEMP.inc.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * @file classes/migration/upgrade/PKPv3_4_0UpgradeMigrationTEMP.inc.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2000-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class PKPv3_4_0UpgradeMigrationTEMP
+ * @brief Temporary, describe upgrade/downgrade operations for those using main branch and 3.4.0.
+ */
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Builder;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Capsule\Manager as Capsule;
+
+class PKPv3_4_0UpgradeMigrationTEMP extends Migration {
```

Comment:
Should `TEMP` be removed here and in the filename?

Reply:
This is just temporary, so I named it so -- the file and the part in the upgrade.xml will then be removed later -- this is just for the developers that are currently working with the main branch and thus already use the version 3.4.0.0, so that they can do the DB upgrade and get the new column...

Reply:
I'm not sure what you mean about temporary. So will this migration be done in 3.4 or not? We've already got a 3.4 migration script at https://github.com/pkp/pkp-lib/blob/main/classes/migration/upgrade/PKPv3_4_0UpgradeMigration.inc.php. Shouldn't this be added there?

Reply:
Yes, it is also there, see above, and it will stay. This is only for us that already use 3.4.0.0 version, to get that DB upgrade too...

Reply:
Oh I see! Sorry for the confusion. I think I would prefer not having this committed at all. I just worry we'll forget to remove it. We can share the SQL command in #pkp-dev for those who need it. @asmecher what do you think?

Reply:
By the way, I'd like to start breaking out the migration scripts into smaller pieces, rather than monolithic classes... that way e.g. they can be run separately using `lib/pkp/tools/migration.php`. I'm thinking that migrations for each version can get their own directory, e.g. `classes/migration/upgrade/3.4.0`. Please feel free to adopt this!

Reply:
@asmecher, so there would be classes/migration/upgrade/3.4.0/ containing e.g. usageStatsTemporaryRecordsMigration.inc.php containing the same code as that PKPv3_4_0UpgradeMigrationTEMP.inc.php, and I would remove that ...TEMP changes and add that usageStatsTemporaryRecordsMigration.inc.php to the current upgrade element in the upgrade.xml, right?

Reply:
Yes, that sounds good, thanks -- but please also include the issue number in the class/filename.